GEON-PPANG-iOS/Global/Utils/LayoutUtils.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LayoutUtils.swift
+//  GEON-PPANG-iOS
+//
+//  Created by JEONGEUN KIM on 2023/09/04.
+//
+
+import UIKit
+
+final class LayoutUtils {
+    
+    class func emptyGroup() -> NSCollectionLayoutGroup {
+        let item = NSCollectionLayoutItem(layoutSize: .init(
+            widthDimension: .fractionalWidth(1),
+            heightDimension: .fractionalHeight(1)
+        ))
+        
+        let groupSize = NSCollectionLayoutSize(
+            widthDimension: .fractionalWidth(1),
+            heightDimension: .fractionalHeight(1)
+        )
+        
+        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitems: [item])
+        return group
+    }
+    
+    class func section(_ exists: Bool, _ size: CGFloat? = 0) -> NSCollectionLayoutSection {
+        
+        let group = LayoutUtils.emptyGroup()
+        let section = NSCollectionLayoutSection(group: group)
+        
+        if exists {
+            section.boundarySupplementaryItems = [LayoutUtils.header(size ?? 0)]
+        }
+        
+        return section
+    }
+    
+    class func header(_ headerSize: CGFloat) -> NSCollectionLayoutBoundarySupplementaryItem {
+        
+        let headerSize = NSCollectionLayoutSize(
+            widthDimension: .fractionalWidth(1.0),
+            heightDimension: .absolute(headerSize))
+        let header = NSCollectionLayoutBoundarySupplementaryItem(
+            layoutSize: headerSize,
+            elementKind: UICollectionView.elementKindSectionHeader,
+            alignment: .top)
+        
+        return header
+    }
```

Comment:
```suggestion
    class func header(ofHeight height: CGFloat) -> NSCollectionLayoutBoundarySupplementaryItem {
        
        let headerSize = NSCollectionLayoutSize(
            widthDimension: .fractionalWidth(1.0),
            heightDimension: .absolute(height))
        let header = NSCollectionLayoutBoundarySupplementaryItem(
            layoutSize: headerSize,
            elementKind: UICollectionView.elementKindSectionHeader,
            alignment: .top)
        
        return header
    }
```

이것도 이런 식으로 쓰면 어떨까여 ??

Reply:
d2d71fefd6f2937fb9604e96d094b6c117a133d0